### DISCUSSION TITLE
This guide vs supplementary guide?

### DISCUSSION THREAD

**Discussion by elysdir** [StyleGuides/WritingStyleGuide#368] [Q&A] [answered]
Hi, folks—I was recently told (by an anonymous source) that this guide was obsolete and has been superseded by the “[Red Hat supplementary style guide for product documentation](https://redhat-documentation.github.io/supplementary-style-guide/).” But it looks like this guide was updated pretty recently, and I’m not seeing anything about it being superseded.

So could you clarify the relationship between this guide and the supplementary guide? Are they both still in use? If so, are they used for different things?

thanks,

—Jed

**Comment by julian-cable** [comment, marked as answer]
Hi Jed, 
Sorry for the delay in replying. Both guides are still used, for different purposes: 
- The Red Hat supplementary style guide is used for product documentation.
- This guide (Red Hat Technical Writing Style Guide on stylepedia.net) is used for all other technical documentation, including training and certification content. 
In both cases, the listed guide acts as the primary style guide, and IBM Style is used for all topics not covered in the primary guide.
